### DISCUSSION TITLE
Arrow Keys To Move Selected Vertices

### DISCUSSION THREAD

**Discussion by fweth** [SVG-Edit/svgedit#745] [New feature request]
It would be cool if one could use the arrow keys to also move a selection of vertices, not just the whole shape.

**Comment by jfhenon** [comment]
Sorry, could you explain a little more?

**Reply by fweth** [reply]
Hmm, I tried again and now it works differently from what I remembered.  I'm talking about the situation where you have a shape and a selection of vertices on that shape, you can use the mouse to move the selected vertices, but apparently you can't use the arrow keys at all.  Maybe I confused this with Boxy SVG, there you can use the keys but the whole shape moves, anyway, it would be nice to move a selection of vertices with the arrow keys as well, not just the mouse.
